comfy_cli/cancellation.py

```
@@ -0,0 +1,117 @@
+        if fire:
+            try:
+                callback()
+            except Exception:  # noqa: BLE001
+                pass
+
+
+_TOKEN: CancellationToken | None = None
+_INSTALLED = False
+
+
+def get_token() -> CancellationToken:
```

Pylint (4.0.5)

[convention] Missing function or method docstring

(C0116)